### ISSUE TITLE
Improvements to cover art enumeration

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#680]
Currently this is a simple div enumeration. There is room for consideration as to whether it should be changed as a linked navigation without a visual change. It affects the behavior of screen readers.

**Comment by tesshucom** [comment]
It's expired. It seems that there is no particular problem at present.

No minor changes to minor web pages are currently planned. 

 - Certain screens like Search, Playlists, and Podcasts may change as the feature expands. In the long run, there is motivation and some value in fixing them.
 - Smaller, and less feature-related fixes are avoided. I haven't had the time to do something like that in the last few years. will continue to be so
 - If you are particular about details, I think you should renovate.
